datacats/userprofile.py

```diff
@@ -131,7 +131,6 @@ def create(self, environment, target_name, stream_output=None):
         remote_server_command(
             ["ssh", environment.deploy_target, "create", target_name],
             environment, self,
-            stream_output=stream_output,
```

Comment:
I thought we agreed that `remotecommand` is only going to return a JSON object that specifies the result in response.
Like:
```json
{
 "status": fail,
 "message": "A site with such a name already exists"
}
```
and we would not want to just print everything we receive to the user. What are the cases for which you want it left?